Reference/using-ioc.md

```diff
@@ -4,7 +4,7 @@ _This section will show you how to setup Ioc/Dependency Injection with your Umbr
 
 ##Overview
 
-We don't use IoC in the Umbraco source code. This isn't because we don't like it or don't want to use it, it's because we want you as a developer to be able to use whatever IoC framework that you would like to use without jumping through any hoops. With that said, it means it is possible to implement whatever IoC engine that you'd like!
+Since Umbraco 7 we use IoC in the Umbraco source code for the backend site. With that said, it is possible to implement whatever IoC engine that you'd like you just need to re-register our api controllers!
```

Comment:
If you revert this description, the other changes below for registering api controllers is fine.